tornjak-frontend/src/tables/clusters-list-table.tsx

```diff
@@ -7,16 +7,47 @@ import {
     clustersListUpdateFunc
 } from 'redux/actions';
 import Table from './list-table';
+import { ClustersList } from "components/types";
+import { DenormalizedRow } from "carbon-components-react";
+import { RootState } from "redux/reducers";
 
 // ClusterListTable takes in 
 // listTableData: clusters data to be rendered on table
 // returns clusters data inside a carbon component table with specified functions
-class ClustersListTable extends React.Component {
-    constructor(props) {
+
+type ClustersListTableProp = {
+    // dispatches a payload for list of clusters with their metadata info as an array of ClustersList Type and has a return type of void
+    clustersListUpdateFunc: (globalClustersList: ClustersList[]) => void,
+    // data provided to the clusters table
+    data: {
+        key: string,
+        props: { cluster: ClustersList }
+    }[] | string | JSX.Element[],
```

Comment:
ditto with other instances

Reply:
Same here. This will be specifically typed when working on https://github.com/spiffe/tornjak/issues/57